### DISCUSSION TITLE
how to enable NodePort on k3s?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [k3s-io/k3s#7126] [General]
how to enable NodePort on k3s?

**Comment by brandond** [comment]
Enable it how? K3s doesn't work any different than any other Kubernetes distro. Is there an issue you'd like to report, or are you just not sure how to use NodePort in general?

**Reply by martinenkoEduard** [reply]
I am not sure how to use  how to use NodePort in general

**Reply by brandond** [reply]
I would read the Kubernetes documentation then.

https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types
